package/package.json

```diff
@@ -20,6 +20,5 @@
     ],
     "website": "https://docs.mesosphere.com/current/usage/service-guides/spark/",
     "selected": true,
-    "version": "$version",
-    "minDcosReleaseVersion": "1.7"
+    "version": "{{package-version}}"
```

Comment:
Stub universes are v2, that param isn't supported in them. The release-to-universe script adds it back (default min version 1.7 or can be configured)